core/server/services/settings/settings-bread-service.js

```diff
@@ -152,6 +152,8 @@ class SettingsBREADService {
             });
         }
 
+        await this.SettingsModel.edit(filteredSettings, options);
```

Comment:
What's the purpose of this additional call as well as the one below?

Reply:
Oh snap, this is an error. I first had the browse one over here, changed it again and forgot to remove this one. Good catch!

Reply:
The reason for the extra browse call above is explained in the new comment line and PR description, being that we need to return all settings when editing a setting.